src/test/java/seedu/address/logic/commands/contacts/FindCommandTest.java

```diff
@@ -5,10 +5,10 @@
 import static org.junit.Assert.assertTrue;
 import static seedu.address.commons.core.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalAddressBook.getTypicalAddressBook;
 import static seedu.address.testutil.TypicalPersons.CARL;
```

Comment:
Are these sample typical persons still needed or should they be moved into `TypicalAddressBook` too?

Reply:
I thought maybe we should keep it for the sake of namespacing `TypicalPersons` vs `TypicalTasks`